Source/VMware.vSphereDSC/VMware.vSphereDSC.build.ps1

```diff
@@ -204,6 +204,44 @@ function Update-ContentOfModuleFile {
     }
 }
 
+function Update-ModuleVersion {
+    [CmdletBinding()]
+    [OutputType([System.Object[]])]
+    param(
+        [System.Object[]] $FileContent
+    )
+
+    $moduleVersionPattern = '*ModuleVersion*'
```

Comment:
You can use  regex to update the build number

$moduleVersionMatch = (get-content <manifest>) | Select-String "(?<=ModuleVersion = ')(\d*\.\d*\.\d*\.\d*)"
$currentVersion = $moduleVersionMatch.Matches[0].Value

$newVersionString =  (new-object "version" $v.Major, $v.Minor, $v.Build, ($v.Revision + 1)).ToString()
(get-content <manifest>) -replace  "(?<=ModuleVersion = ')(\d*\.\d*\.\d*\.\d*)", $newVersionString | Set-Content